lib/contestant.rb

```diff
@@ -0,0 +1,30 @@
+class Contestant
+  attr_reader :first_name,
+              :last_name,
```

Comment:
Since we don't ever call the methods `.first_name` and `.last_name` these do not need to be attr_readers.